### PR TITLE
Remove useless promise

### PR DESCRIPTION
**Pull request by jrichter1**
AFAIK it does nothing apart from making some unit tests fail on windows.

### CODE REVIEW

File: browser/model/virtualbox.js [hunk 1]
```diff
@@ -64,9 +64,7 @@ class VirtualBoxInstall extends InstallableItem {
                 resolve(output);
             });
           } else {
-            return new Promise((resolve, reject) => {
```
Comment (dgolovin) [inline browser/model/virtualbox.js]: I remember this was exactly why detection for virtualbox didn't work for Nick.

Comment (jrichter1) [inline browser/model/virtualbox.js]: I believe this is not the one Nick introduced - that one is still there (and it is commented).
See https://github.com/jrichter1/developer-platform-install/blob/promise-fix/browser/model/virtualbox.js#L82

Comment (dgolovin) [inline browser/model/virtualbox.js]: I see the difference now, that one was inside that call. Current one is already inside Promise instance. You are right.

**Comment by dgolovin** [comment]
+1 to merge

**Comment by jrichter1** [comment]
Pushed to master.

**Pull request closed without merging**